.cirrus.yml

```diff
@@ -0,0 +1,207 @@
+#
+# Notes
+#
+# Cirrus CI has a concurrency limit of 24 CPU cores for open source projects.
+# This means we should not have more than 12 pipelines with more than 2 CPUs
+# in each pipeline running parallel
+#
+# Documentation:
+# - https://cirrus-ci.org/examples/
+# - https://cirrus-ci.org/guide/writing-tasks/
+#
+
+#
+# Templates
+#
+
+---
+
+elk_common_cpu_and_memory_config_for_build_and_test_template: &ELK_COMMON_CPU_AND_MEMORY_CONFIG_FOR_BUILD_AND_TEST
+  cpu: 2
+  memory: 4GB
+
+elk_common_build_and_test_debug_template: &ELK_COMMON_BUILD_AND_TEST_DEBUG
+  env:
+    RUSTDOCFLAGS: -D warnings
+  timeout_in: 10m
+  build_script: cargo build --workspace --all --examples
```

Comment:
I found out that benchmarks are also not included in `--all` could you add also `--benches` even when we do not yet have them. Then we cannot forget this in the future.